didact/src/components/courses/DetailedCourse.jsx

```diff
@@ -46,7 +46,7 @@ const DetailedCourse = props => {
         setLessonExpanded(isExpanded ? panel : false)
     }
 
-    if (course && sections) {
+    if (!state.coursesReducer.isLoading && (course && sections)) {
```

Comment:
Checks isLoading from state, and makes sure that there is a course and sections are there before loading